sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpReceiver.cs

```diff
@@ -241,8 +241,8 @@ private static IReadOnlyList<ServiceBusReceivedMessage> EmptyList
                 // to log here.
                 throw;
             }
-            catch (TaskCanceledException)
-                when (_isProcessor)
+            catch (OperationCanceledException)
```

Comment:
Good call; with the AMQP lib support, we can no longer count on the specific task exception.   I'll see if we have any similar blocks in EH that need this kind of update.